extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -396,6 +419,208 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
+			if (!this.viewInfo.loginNames?.options.find(owner => owner === this.objectInfo.owner)) {
+				loginNames[0] = this.objectInfo.owner;
+			} else {
+				loginNames.splice(0, 1);
+			}
+			let ownerDropbox = this.createDropdown(localizedConstants.OwnerText, async () => {
+				this.objectInfo.owner = ownerDropbox.value as string;
+			}, loginNames, this.objectInfo.owner);
+			containers.push(this.createLabelInputContainer(localizedConstants.OwnerText, ownerDropbox));
+		}
+
+		// This check box is checked and disabled because full-text indexing is always enabled in SQL Server
+		const useFullTextIndexing = this.createCheckbox(localizedConstants.UseFullTextIndexingText, async () => { }, this.objectInfo.fullTextIndexing, false);
+		containers.push(useFullTextIndexing);
+
+		return this.createGroup('', containers, false);
+	}
+
+	private initializeDatabaseFilesSection(): azdata.GroupContainer {
+		this.databaseFilesTable = this.modelView.modelBuilder.table().withProps({
+			columns: [{
+				type: azdata.ColumnType.text,
+				value: localizedConstants.LogicalNameText
+			}, {
+				type: azdata.ColumnType.text,
+				value: localizedConstants.FileTypeText
+			}, {
+				type: azdata.ColumnType.text,
+				value: localizedConstants.FilegroupText
+			}, {
+				type: azdata.ColumnType.text,
+				value: localizedConstants.SizeInMbText
+			}, {
+				type: azdata.ColumnType.text,
+				value: localizedConstants.AutogrowthMaxsizeText
+			}, {
+				type: azdata.ColumnType.text,
+				value: localizedConstants.PathText
+			}, {
+				type: azdata.ColumnType.text,
+				value: localizedConstants.FileNameText
+			}],
+			data: this.objectInfo.files?.map(file => {
+				return this.convertToDataView(file);
+
+			}),
+			height: getTableHeight(this.objectInfo.files?.length, DefaultMinTableRowCount, DefaultMaxTableRowCount),
+			width: DefaultTableWidth,
+			forceFitColumns: azdata.ColumnSizingMode.DataFit,
+			CSSStyles: {
+				'margin-left': '10px'
+			}
+		}).component();
+		const databaseFilesButtonContainer = this.addButtonsForTable(this.databaseFilesTable, localizedConstants.AddButton, localizedConstants.RemoveButton,
+			(button) => this.onAddDatabaseFilesButtonClicked(button), (button) => this.onEditDatabaseFilesButtonClicked(button), localizedConstants.EditButton, () => this.onRemoveDatabaseFilesButtonClicked());
+
+		return this.createGroup(localizedConstants.DatabaseFilesText, [this.databaseFilesTable, databaseFilesButtonContainer], true);
+	}
+
+	/**
+	 * Converts the file object to a data view object
+	 * @param file database file object
+	 * @returns data view object
+	 */
+	private convertToDataView(file: DatabaseFile): any[] {
+		return [
+			file.name,
+			file.type,
+			file.fileGroup,
+			file.sizeInMb,
+			file.isAutoGrowthEnabled ? localizedConstants.AutoGrowthValueStringGenerator(file.type !== this.viewInfo.fileTypesOptions[2]
+				, file.autoFileGrowth.toString()
+				, file.autoFileGrowthType === localizedConstants.PercentText
```

Comment:
Shouldn't autoFileGrowthType not be localized? You should be defining a set of constant non-localized values that it can be and comparing against that.

Reply:
with using the enum, no need of localization required for this property. updated the changes. thanks